worker-queue-rabbit/src/main/java/com/github/workerframework/queues/rabbit/RabbitWorkerQueueConfiguration.java

```diff
@@ -50,6 +50,12 @@ public class RabbitWorkerQueueConfiguration
     @NotNull
     @Size(min = 1)
     private String inputQueue;
+    /**
+     * The queue to send notifications of missing offloaded task data to.
+     */
+    @NotNull
```

Comment:
You're making this mandatory!?  This would mean that every worker has to be updated with a value for this when moving to this version, even if offloading is not enabled.

This was discussed before for the last configuration entry added: https://github.com/WorkerFramework/worker-framework/pull/214#discussion_r2127335507

Reply:
@dermot-hardy This now allows nulls, and is defaulted in the code.  This removes the need for the other pull requests now since they no longer need to be updated to include the missing queue cfg.  I had assumed that the default-config was used for all workers, which meant that there was no need to update the worker configuration at all.